frontend/src/components/worksheets/Worksheet/WorksheetHeader.js

```diff
@@ -114,6 +114,17 @@ export default ({
                                             />
                                         </div>
                                     </Popover>
+                                    &nbsp;tags:&nbsp;
+                                    <div style={{ display: 'inline-block' }}>
+                                        <WorksheetEditableField
+                                            canEdit={canEdit}
+                                            dataType='list'
+                                            fieldName='tags'
+                                            value={info.tags.join(' | ')}
```

Comment:
I think we wanted to remove the bar between the tags and treat space as the delimiter?

Reply:
true...